concourse/tasks/gcloud-get-secret.yaml

```diff
@@ -0,0 +1,30 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
+    tag: alpine
+
+params:
+  secret-version: "latest"
```

Comment:
since these are environment variables we should use underscores and capital letters, so maybe `SECRET_VERSION` or just `VERSION` since it's internal to this task

Reply:
Changed to SECRET_VERSION and SECRET_NAME